services/DBApi/databases/libs/query-format.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * @function getQueryFormat(...) transforms a key-value dictionary into a valid SQL Query for GET requests
+ * 
+ * @param {JSON} queryParams    : a dictionary where keys are column names and values are entries of the columns
```

Comment:
What is the format of queryParams in URL form
e.g. localhost:3000/dbname/tablename?[WHAT GOES HERE]

Reply:
localhost:3000/dbname/tablename?username=someuser&password=somepw

=> queryParams will be { username: "someuser", password: "somepw" }

Reply:
How about for POST, PUT and DELETE? I get some errors with the above in postman

Reply:
Under Body, select raw and JSON (application/json)

For POST and DELETE, the keys are columns and values are entries, e.g.
{ "username": "someuser", "password": "somepw" }

For PUT, it is a '2-dimensional' json, you need to specify what are the updates and what are the conditions, e.g.
{ "updates": { "password": "newpw", "email": "newemail" }
  "conditions": { "username": "onlyupdatethisuser" } }